src/components/CorrectionReviewBoxes/CorrectionReviewBoxes.scss

```diff
@@ -23,11 +23,6 @@
     margin-top: 0.33333334rem;
 
     .correction-difficulty {
-      background-color: #19ba3f62;
-      border: 0.06666667rem solid #19ba3f;
-      border-radius: 0.8666667rem;
-      font-size: 1.0666667rem;
-      height: 1.6666666rem;
       text-align: center;
```

Comment:
muss das hier noch drin bleiben? das sollte doch eigentlich in `DifficultyLabel` drin sein